src/dotnet/APIView/APIView/HTMLRendererAPIV.cs

```diff
@@ -31,17 +63,24 @@ protected override void RenderNewline(StringBuilder builder)
 
         protected override void RenderSpecialName(StringBuilder builder, string word)
         {
-            builder.Append("<span class=\"specialName\">").Append(word).Append("</span>");
+            builder.Append("<span class=\"specialName\">").Append(EscapeHTML(word)).Append("</span>");
         }
 
         protected override void RenderType(StringBuilder builder, string word)
         {
-            builder.Append("<span class=\"type\">").Append(word).Append("</span>");
+            var shortName = word.Substring(word.LastIndexOf(".") + 1);
```

Comment:
Is this still used?

Reply:
Nope - can be tossed. Thanks for catching!